src/os/bluestore/BlueFS.cc

```diff
@@ -2443,38 +2447,17 @@ void BlueFS::_pad_bl(bufferlist& bl)
   }
 }
 
-
-int BlueFS::_flush_and_sync_log(std::unique_lock<ceph::mutex>& l,
-				uint64_t want_seq,
-				uint64_t jump_to)
+// Adds to log_t file modifications mentioned in `dirty_files`.
+// Note: some bluefs ops may have already been stored in log_t transaction.
+uint64_t BlueFS::_consume_dirty()
 {
-  while (log_flushing) {
```

Comment:
Don't we still need for proper waiting on 'log_flushing'? If not then we should cleanup that stuff totally (remove log_flashing member and its usage). I can still see waiting on this flag in _compact_log_async but there is no setting it to true any more